gatsby-node.js

```diff
@@ -84,6 +92,76 @@ const constructDbFilter = () => ({
     commit_hash: process.env.COMMIT_HASH || { $exists: false },
 });
 
+const createTagPageType = async (createPage, pageMetadata, stitchType) => {
+    const isAuthor = stitchType === 'author';
+    const pageType = STITCH_TYPE_TO_URL_PREFIX[stitchType];
+    const res = await stitchClient.callFunction('getValuesByKey', [
```

Comment:
Looking really good! Can you explain what this stitch call in conjunction with line line `108` is doing? I know you explained some of it in person but now that the code is here things will sink in more for me. I haven't touched any of the stitch stuff yet so curious to know how these are working!

Reply:
For example,  I see that we get an array of items for this but was I'm wondering why we iterate through each and make other async calls for each item.

Reply:
Sure! So there are two stitch functions we can call to get data: `getValuesByKey` and `fetchDevhubMetadata`.

`getValuesByKey` takes one of our tag types as a parameter (such as `language`) and returns back all of the values for that key in stitch (such as [`node`, `java`, etc.]) as an array of objects.

For each one of these objects (let's say they are languages), we then need to get all of the article pages for that type of language. Now that we know all of the languages that exist, we can call the `fetchDevhubMetadata` on each one to return to us the article pages associated with it.

You can think of it like a query for what languages exist and then a map to its pages, does that help?

Reply:
Ah I see. Yea this was great, thanks!